crates/net/network/src/transactions/mod.rs

```diff
@@ -972,6 +983,8 @@ where
             return
         }
 
+        let mut transactions = transactions.0;
```

Comment:
Should we impl deref etc on `PooledTransactions`?

Reply:
yeah sure, `Deref` is always safe, `DerefMut` you have to be sure this doesn't effect existing behaviour. usually only do it on my own new types cause of this.

Reply:
anyhow, out of scope of pr I think